.github/scripts/test_sanger_profile.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+set -eo pipefail
+export PATH=/opt/conda/bin:$PATH
+
+# test --sanger profile
+# there are only 2 available cpus in the github runner execution
+sed -i s'/cpus = 4/cpus = 2/'g conf/coguk/sanger.config
+echo run pipeline with sanger profile >> artifacts/test_artifact.log
+NXF_VER=20.03.0-edge nextflow run ./main.nf \
+       -profile sanger,singularity \
+       --directory $PWD/.github/data/ \
+       --illumina \
+       --prefix test
+cp .nextflow.log ./artifacts/sanger.profile.nextflow.log
+
+# check that sanger profile activated 4 cpus on bwa mem:
+find work -name .command.err \
+    | xargs cat | grep '\[main\] CMD: bwa mem -t 2' \
+    && echo "sanger profile: bwa started with 4 cpus" \
+	|| exit 1
+
+# check that sanger profile activated params.allowNoprimer = false:
+find work -name .command.sh \
+    | xargs cat | grep 'ivar trim -e' \
+    && exit 1 \
+	|| echo "sanger profile: did NOT ran ivar trim -e" 
+
+# clean-up for following tests
+mv results results_sanger_profile
```

Comment:
Is this intended, or copy/paste error? Generally, do any of the tests look at the output of other tests? Does the results dir need to be kept for any of the tests? I think only my suggested way of testing `--ref` needs prior output, and that just needs specific output files.

Reply:
ok, I will remove the results instead of renaming them.